src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.*;
+import tasks.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greeting = " Hello! I'm Duke\n" + " What can I do for you?";
+        String bye = "Bye. Hope to see you again soon!";
+        System.out.println(greeting);
+
+        boolean running = true;
+        String input;
+        Scanner sc = new Scanner(System.in);
+        List<Task> list = new ArrayList<>();
+        while(running){
+            input = sc.nextLine();
+            if(input.matches("bye")){
+                break;
+            }else{
+                controller(input,list);
+            }
+
+        }
+
+        System.out.println(bye);
+
     }
+
+    //Will be refactored and in better components
+    public static void controller(String input, List<Task> list ){
+        int index = 0;
+        String secPart;
+        String description;
+        if(input.matches("list")){
+            for(int i = 0; i < list.size(); i++){
+                System.out.println(i + 1 + ". " + list.get(i).toString());
+            }
+        }else if(input.contains("done")){
+            int option = Integer.parseInt(input.replaceAll("done","").trim());
+            if(option > 0 && option <= list.size()){
+                list.get(option - 1).changeDoneTo(true);
+                System.out.println("Nice! I've marked this task as done:");
+                System.out.println(" " + list.get(option-1).toString());
+            }
+
+
+        }else if(input.contains("todo")){
+            description = input.replaceFirst("todo","").trim();
+            list.add(new ToDo(description));
+            System.out.println("Got it. I've added this task:");
+            System.out.println(list.get(list.size()-1).toString() );
+            System.out.println("Now you have " + list.get(0).getCumulatedTasksAdded() + " tasks in the list.");
```

Comment:
Line 49 to 51. can improve by putting into a method as it is repeated for todo, deadline and event